site/content/en/docs/Reference/Networking/proxy.md

```diff
@@ -0,0 +1,109 @@
+export NO_PROXY=localhost,127.0.0.1,10.96.0.0/12,192.168.99.0/24,192.168.39.0/24
+
+minikube start
+```
+
+To make the exported variables permanent, consider adding the declarations to ~/.bashrc or wherever your user-set environment variables are stored.
+
+### Windows
+
+```shell
+set HTTP_PROXY=http://<proxy hostname:port>
+set HTTPS_PROXY=https://<proxy hostname:port>
+set NO_PROXY=localhost,127.0.0.1,10.96.0.0/12,192.168.99.1/24,192.168.39.0/24
+
+minikube start
+```
+
+To set these environment variables permanently, consider adding these to your [system settings](https://support.microsoft.com/en-au/help/310519/how-to-manage-environment-variables-in-windows-xp) or using [setx](https://stackoverflow.com/questions/5898131/set-a-persistent-environment-variable-from-cmd-exe)
+
+## Configuring Docker to use a proxy
+
+As of v1.0, minikube automatically configures the Docker instance inside of the VM to use the proxy environment variables, unless you have specified a `--docker-env` override. If you need to manually configure Docker for a set of proxies, use:
+
+```shell
+minikube start \
+  --docker-env=HTTP_PROXY=$HTTP_PROXY \
+  --docker-env HTTPS_PROXY=$HTTPS_PROXY \
+  --docker-env NO_PROXY=$NO_PROXY
+```
+
+## Troubleshooting
+
+### unable to cache ISO... connection refused
+
+```text
+Unable to start VM: unable to cache ISO: https://storage.googleapis.com/minikube/iso/minikube.iso:
+failed to download: failed to download to temp file: download failed: 5 error(s) occurred:
+
+* Temporary download error: Get https://storage.googleapis.com/minikube/iso/minikube.iso:
+proxyconnect tcp: dial tcp <host>:<port>: connect: connection refused
+```
+
+This error indicates that the host:port combination defined by HTTPS_PROXY or HTTP_PROXY is incorrect, or that the proxy is unavailable.
+
+## Unable to pull images..Client.Timeout exceeded while awaiting headers
+
+```text
+Unable to pull images, which may be OK:
+
+failed to pull image "k8s.gcr.io/kube-apiserver:v1.13.3": output: Error response from daemon:
+Get https://k8s.gcr.io/v2/: net/http: request canceled while waiting for connection
+(Client.Timeout exceeded while awaiting headers)
+```
+
+This error indicates that the container runtime running within the VM does not have access to the internet. Verify that you are passing the appropriate value to `--docker-env HTTPS_PROXY`.
+
+## x509: certificate signed by unknown authority
+
+```text
+[ERROR ImagePull]: failed to pull image k8s.gcr.io/kube-apiserver:v1.13.3:
+output: Error response from daemon:
+Get https://k8s.gcr.io/v2/: x509: certificate signed by unknown authority
+```
+
+This is because minikube VM is stuck behind a proxy that rewrites HTTPS responses to contain its own TLS certificate. The [solution](https://github.com/kubernetes/minikube/issues/3613#issuecomment-461034222) is to install the proxy certificate into a location that is copied to the VM at startup, so that it can be validated.
```

Comment:
... is to install the proxy certificate at the location copied to the VM when starting up (?)

Reply:
maybe break this into two sentences, too much all in one go